pc/user.hpp

```diff
@@ -43,7 +43,8 @@ namespace pc
     // symbol price schedule callback
     void on_response( price_sched *, uint64_t ) override;
 
-    // send all pending updates
+    // send a batch of pending price updates. This function eagerly sends any complete batches.
```

Comment:
Might be worth clarifying the semantics:

`// If there are multiple complete batches, only one will be sent.`

Reply:
I assume those are the desired semantics, as the aim of this is to stagger transactions.